imls-directus/init/config.sql

```diff
@@ -0,0 +1,10 @@
+CREATE SCHEMA api;
```

Comment:
Probably also should go into the `imls-backend` (or whatever name we pick) directory?